### ISSUE TITLE
Fail case if location isn't ready

### ISSUE THREAD

**Issue by nighelles** [remulasce/metroapp#255]
If there isn't a location available, what is the desired fail case for getcurrentdistancetostop?

0? 1000?

**Comment by remulasce** [comment]
There might be some checks for -1 somewhere.

**Comment by nighelles** [comment]
Hmm.

**Comment by remulasce** [comment]
Yeah, Android location provider returns -1 if the stop or user has no location.
